Procfile

```diff
@@ -1 +1 @@
-worker: node src/spawn_tests.js
+worker: npm run compile && node lib/main.js test
```

Comment:
I  think it should be fine doing it the normal way. If you see problems doing it that way lets switch it, but for now keep it like everything else.